### DISCUSSION TITLE
space bandits

### DISCUSSION THREAD

**Discussion by mmails** [unitystation/unitystation#7083] [Ideas]
### Description

Space Bandits

Their job is to rob the on-station vault at all costs, they spawn in a shuttle a few thousand tiles away from the station. They could be an admin spawn or a normal antag.

**Comment by corp-0** [comment]
I'm making this a discussion instead of issue so the idea can mature and find proper requirements for it to be fulfilled.

**Comment by VictorVow** [comment]
This doesn't sound very different from Space Pirates on tg.
